dellyGermline.wdl

```diff
@@ -257,6 +317,161 @@ task filter{
   }
 
   output {
-    File germlineVcf = "germline.vcf"
+    File germlineSVVcf = "germlineSV.vcf"
+    File germlineSVBcf = "germlineSV.bcf"
```

Comment:
I wonder if we need both a bcf and vcf file. I originally converted to vcf just because it was the end of the pipeline at the time, and I figured I'd make it human readable. Now that we need the bcf for CNV calling, maybe the outputs from the two filter tasks can just be the bcfs, and we can remove the vcf conversion lines? @bibilujan do you have any thoughts on this?

Reply:
To be consistent with other workflows, final output should be a vcf, and the bcf is needed for CNV calling. So its fine to leave as is!